dataverse-webapp/src/main/webapp/resources/vecler/theme.scss

```diff
@@ -2454,7 +2514,1084 @@ div.ui-dialog.largePopUp {
+			}
+		}
+
+		.text-danger {
+			color: $contrast-foreground-color;
+		}
+		.label.label-danger {
+			border: 1px solid $contrast-foreground-color;
+			background: $contrast-foreground-color;
+			color: $contrast-background-color;
+		}
+
+		#navbar-underline {
+			border-color: $contrast-link-color;
+		}
+
+	}
+	.navbar-default {
+		#topNavBar .navbar-nav {
+			a {
+				color: $contrast-link-color;
+	
+				&:hover, &:active, &:focus {
+					color: $contrast-foreground-color;
+					text-decoration: underline;
+				}
+			}
+	
+			> .open > a {
+				background: $contrast-alt-background-color
+			}
+	
+			> li > a .caret, .navbar-nav > form > a .caret {
+				color: $contrast-foreground-color;
+			}
+		}
+
+		.navbar-toggle {
+			&:hover {
+				background: $contrast-hover-color;
+			}
+	
+			.icon-bar {
+				background-color: $contrast-foreground-color;
+			}
+		}
+
+		.navbar-collapse {
+			border-bottom-color: $contrast-link-color;
+		}
+	}
+
+	#footer .footer-links .row {
+		border-top-color: $contrast-link-color;
+
+		div[class^="col-"] p span {
+			border-left-color: $contrast-link-color;
+		}
+	}
+
+	/* Modals */
+
+	#sharrre-block a {
+		border: 1px solid $contrast-link-color;
+		background-color: $contrast-background-color !important;
+		color: $contrast-link-color;
+
+		span {
+			color: $contrast-link-color;
+		}
+	}
+
+	.ui-dialog.ui-widget-content {
+		background-color: $contrast-background-color;
+
+		&, .ui-state-default, .ui-widget-header, .ui-widget-content {
+			.ui-icon {
+				background-image: url($contrast-primefaces-main);
+			}
+
+			button .ui-icon {
+				background-image: url($contrast-primefaces-main-alt);
+			}
+		} 
+
+		.ui-dialog-content, .help-block {
+			color: $contrast-foreground-color;
+		}
+
+		.text-info {
+			color: $contrast-foreground-color;
+		}
+
+		.ui-widget-header,
+		.ui-fileupload-content.ui-widget-content {
+			border-color: $contrast-link-color;
+		}
+	}
+
+	/* Bootstrap tooltips, popovers */
+	
+	.tooltip {
+		.tooltip-inner {
+			background-color: $contrast-foreground-color;
+			color: $contrast-background-color;
+		}
+
+		&.top .tooltip-arrow {
+			border-top-color: $contrast-foreground-color;
+		}
+		&.bottom .tooltip-arrow {
+			border-bottom-color: $contrast-foreground-color;
+		}
+		&.left .tooltip-arrow {
+			border-left-color: $contrast-foreground-color;
+		}
+		&.right .tooltip-arrow {
+			border-right-color: $contrast-foreground-color;
+		}
+	}
+
+	.popover {
+		background: $contrast-background-color;
+		border: 1px solid $contrast-foreground-color;
+
+		&.top > .arrow::after {
+			border-top-color: $contrast-foreground-color;
+		}
+		&.bottom > .arrow::after {
+			border-bottom-color: $contrast-foreground-color;
+		}
+		&.left > .arrow::after {
+			border-left-color: $contrast-foreground-color;
+		}
+		&.right > .arrow::after {
+			border-right-color: $contrast-foreground-color;
+		}
+	}
+}
+
+/* Definitions */
+
+body.high-contrast-yellow-on-black {
+	@include high-contrast-template();
+}
```

Comment:
I'm not sure - but maybe we should indicate somehow readonly textareas
![readonly_textbox](https://user-images.githubusercontent.com/8190961/78793776-29771b80-79b3-11ea-8a93-9b8d714bfd2a.png)

Reply:
IMO - we should have perceptible change in styling when mouse is over arrow
![featured_arrow](https://user-images.githubusercontent.com/8190961/78794162-aaceae00-79b3-11ea-8e13-d23b2aefe6a9.png)

Reply:
we have problems with style for selectcheckboxmenu in edit metadata
![edit_metadata_subject](https://user-images.githubusercontent.com/8190961/78794309-d9e51f80-79b3-11ea-9252-eddcd4e63c4d.png)

Reply:
Fixed